### ISSUE TITLE
Link checking with DocumenterCitations

### ISSUE THREAD

**Issue by ArnoStrouwen** [JuliaDocs/Documenter.jl#2050]
Link checking errors when using citations from
https://github.com/ali-ramadhan/DocumenterCitations.jl

**Comment by mortenpi** [comment]
While we should probably provide some hooks in Documenter for DocumenterCitations to use, this seems more like a feature request for that package?

**Comment by odow** [comment]
Should this be transferred to https://github.com/JuliaDocs/DocumenterCitations.jl? cc @goerz

**Comment by goerz** [comment]
Yes, feel free to open an issue, if you have an example where DocumenterCitations has some unexpected behavior… but I was under the assumption that link checking already works. I'm pretty sure I've seen it warn about broken links, although there may not be an explicit test to ensure that. In any case, the way it should work is that the normal link checking in Documenter just automatically handles any URLs in the rendered bibliography. Since the pipeline step that does the checking runs after any of the pipeline steps that expand the bibliography, it shouldn't be much of a problem.
